EIPS/eip-2612.md

```diff
@@ -1,11 +1,10 @@
 ---
 eip: 2612
-title: "EIP-20 Permit Extension: Signed Approvals"
+title: "Permit Extension for EIP-20: Signed Approval"
```

Comment:
@SamWilsn There may be a problem with the linter's parser. It says the title is too long but it's exactly 44 characters.

Reply:
Just realize there is a quotation pair here. Drop them and it should be good